language/evm/hardhat-examples/contracts/FortyTwo.move

```diff
@@ -0,0 +1,26 @@
+// FortyTwo.move
```

Comment:
You can probably drop this comment -- I included it only because it was part of the tutorial and I needed to tell the reader which file this was.

Reply:
done.